netlogo-gui/src/main/app/codetab/CodeTab.scala

```diff
@@ -50,7 +54,20 @@ class CodeTab(val workspace: AbstractWorkspace) extends JPanel
   override def zoomTarget = text
 
   val errorLabel = new EditorAreaErrorLabel(text)
-  val toolBar = getToolBar
+  val toolBar = new ToolBar {
```

Comment:
It this was the last statement in the initializer, it might not be more cluttered, but there are several statements which follow this. So the comparison is

```scala
val toolBar = new ToolBar {
  // ...
}
val scrollableEditor = // ...
```

```scala
val toolBar = getToolBar
val scrollableEditor = // ...

// ...
def getToolBar = new ToolBar {
  // ...
}
```

I find the second much more reasonable.